aviator.yml

```diff
@@ -1,15 +1,15 @@
 spruce:
-- base: ci/meta.yml
-  prune:
-  - meta
-  merge:
-  - with_in: ci/
-    regexp: ".*yml"
-  - with_in: ci/jobs/
-    except:
-    - management-dev.yml.j2
-    - management.yml.j2
-  to: aviator_pipeline.yml
+  - base: ci/meta.yml
+    prune:
+      - meta
+    merge:
+      - with_in: ci/
+        regexp: ".*yml"
+      - with_in: ci/jobs/
+        except:
+          - management-dev.yml.j2
```

Comment:
Can the regex not be altered to ignore Jinja files?

Reply:
Actually, you can just change the files back to `.yml`, the only things Jinja is putting in is the account numbers, which are already available as variables
https://github.com/dwp/dataworks-admin-utils/blob/master/ci/ami-cleanup/meta.yml#L19

Reply:
I assume they're all in SSM, which isn't configured as a Credentials Manger in AWS-Concourse.  We'd need to transfer the desired content over to SM, to do that, which isn't in the scope of this work.